### ISSUE TITLE
[ACCESSIBILITY] WEB: Checkmark for uploading event image is hard to see

### ISSUE THREAD

**Issue by vburgett** [rokwire/illinois-app#4683]
When uploading an event image on [the web](https://api-dev.rokwire.illinois.edu/illinois/), the checkmark and back arrow are not high contrast enough. Please change the color to white.

This is not an issue in the app/client.

![Image](https://github.com/user-attachments/assets/fa93cda7-0f17-4fd6-bcb1-bb53efca7657)

**Comment by vburgett** [comment]
@TodorBachvarov is this a quick update that can be addressed soon?

**Comment by vburgett** [comment]
Can this ticket get addressed before you update the website on prod? @pmarkhennessy @TodorBachvarov

**Comment by dobromirdobrev** [comment]
@vburgett ,

This is fixed in version 7.1.6 of the web app. 

Please note that this UI is part of external plugin that we use and unfortunately the color of the icons are not customizable.
Thus, we changed the behaviour and instead showing a page, we show a dialog. It clearly shows all buttons etc.

Please test when you have a chance and let us know if this works. Thanks.

**Comment by vburgett** [comment]
This new window works well on the web. Thank you, @dobromirdobrev! 

If I tested correctly, the update does not affect the mobile app—is that correct?

Is this a new plugin or the same plugin? If it's new, can we change the colors as outlined in the below screenshot?

Navy: button text, slider circle
White: button background color
Grey: slider bar

![Image](https://github.com/user-attachments/assets/f1ef4c1c-98ee-4e09-ae01-7a859bc52766)

**Comment by dobromirdobrev** [comment]
@vburgett ,

> If I tested correctly, the update does not affect the mobile app—is that correct?

Correct. The plugin that we use in the mobile app is different than the plugin that we use in the web app.
This is because the mobile app plugin does not work on web.

> Is this a new plugin or the same plugin? If it's new, can we change the colors as outlined in the below screenshot?
> 
> Navy: button text, slider circle
> White: button background color
> Grey: slider bar

This is the same plugin. I just changed the view - from page to dialog. This behaviour is supported by the plugin.
Unfortunately, these colors that you pointed in the screenshot are not customizable and we cannot change them.
